scripts/joinmarket-qt.py

```diff
@@ -1144,6 +1144,46 @@ def validateSingleSend(self):
                 "Amount, in bitcoins, must be provided.",
                 mbtype='warn', title="Error")
             return False
+        if len(self.changeInput.text()) != 0:
+            change_addr = self.changeInput.text()
+            valid, errmsg = validate_address(change_addr)
+            if not valid:
+                JMQtMessageBox(self,
+                               "Custom change address is invalid: \"%s\"" % errmsg,
+                               mbtype='warn', title="Error")
+                return False
+
+            msg = """You are attempting to send change to a custom change address.
+Change outputs are usually directly linkable to your CoinJoin inputs,
+and incautious combination of custom change UTXOs can catastrophically
```

Comment:
This seems like a good warning text, however, it'll be pretty annoying to see it in every single case? I would even tend to argue that this one isn't needed (unlike the following one, which I think *is* needed), but I do think it's fine to have it.

Reply:
I agree it's not critical. But it can't hurt to warn the user to be careful. And hopefully the user is not doing this very frequently or else they risk getting fingerprinted. So it shouldn't be too annoying.

Reply:
Maybe could be removed for direct send (i.e., 0 counterparties), it's mostly a CoinJoin specific warning.
Although, I think it's an important message, so maybe, even if users are not coinjoining, it's good for them to see it and learn about it.

Reply:
Good point, this should be removed I agree, because the text as-is will be just wrong (referring to the tx as a coinjoin).